.github/CODEOWNERS

```diff
@@ -4,7 +4,9 @@
 # For syntax help see:
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
-*                       @googleapis/yoshi-java
+# The @googleapis/googleapis-auth is the default owner for changes in this repo
+*                       @googleapis/yoshi-java @googleapis/googleapis-auth
+**/*.java               @googleapis/googleapis-auth
```

Comment:
dumb q- what does line 9 do? Seems like line 8 is giving `@googleapis/googleapis-auth` edit access to all the files in the repo already, so is line 9 redundant?

Reply:
I believe from the docs (https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#example-of-a-codeowners-file), the last matching pattern takes precedence. From any non *.java files, yoshi-java and googleapis-auth should be able to approve, but only googleapis-auth is allowed to approve for *.java. 

These changes are from owlbot (and I believe from the template in the synthtool).

Reply:
I think that is good config, so infra PR aren't blocked by auth team